docs/dashboard_config.md

```diff
@@ -82,11 +82,10 @@ notebookSizes:
 
 ### Notebook controller
 
-The `notebookController` field controls the Notebook Controller options such as whether it is enabled in the dashboard and which parts should be visible.
+The `notebookController` field controls the Notebook Controller options such as which parts should be visible.
 
 ```yaml
 notebookController:
-  enabled: true
```

Comment:
@andrewballantyne @DaoDaoNoCode same here in the documentation. notebookController.enabled stays in, even though it really in terms of semantics means Jupyter tile and links to it visible true or false.